### DISCUSSION TITLE
Users Page does not show

### DISCUSSION THREAD

**Discussion by michaelswells** [alirezanet/InfiniteEnumFlags#3] [General]
I was running your example (which I did not modify) and did the following:
1. Create new user viewer@localhost
2. Create new user manager@localhost
3. login as admin
4. create new role Viewer
5. create new role Manager
6. assign ViewUsers permission to role Viewer
7. assign ManageUsers permission to role Manager
8. Login as viewer@localhost => Should see menu item for Users
9. Login as manager@localhost => Should see menu item for Roles

I was not able to see the expected menu items. I believe your encoding/decoding and bit-arithmetic is not working properly.
I do not know where to begin debugging InfiniteEnums. The FlexAuth program from Jason did everything as expected though.

**Comment by alirezanet** [comment]
Hi @michaelswells,
Good catch ... 

The problem probably is related to the `FlexibleAuthorizeView`, right now it is only working if the user has both permissions!!
![image](https://user-images.githubusercontent.com/7004080/205205322-649d60e4-b31c-47e5-8b3f-b6cd98a6a819.png)

Or might be related to blazor/wasm limitations.


For further investigation, I created an issue for this (#4)

thank you

**Comment by alirezanet** [comment]
Problem (#3 & #4) is solved in `v0.4.0`
Thank you for the feedback
